### DISCUSSION TITLE
"Cannot run texdoc: spawn texdoc ENOENT"

### DISCUSSION THREAD

**Discussion by asimenio-asteri** [James-Yu/LaTeX-Workshop#3144] [Q&A]
How would I fix this error?
I am running the Windows 10, and vscode ver. 1.42.3, extension v8.8.0
I have tried 2 things that didn't work:
try viewing the pdf for a chemistry molecule diagram w/ package stuff
try viewing a normal latex document
The console says it can't find the PDF file.

**Comment by jlelong** [comment]
Please post the complete logs of the extension otherwise we cannot help

![74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524](https://user-images.githubusercontent.com/2910140/77959162-6f0e5700-72d6-11ea-9fe0-e758333028d9.png)

**Comment by asimenio-asteri** [comment]
compiler: nothing
workshop: https://pastebin.com/58r5HZkz

**Comment by jlelong** [comment]
It looks like you do not have a LaTeX distribution installed on your computer or at least none of the LaTeX compilers are in your `PATH`.

**Comment by asimenio-asteri** [comment]
Ok, i will install it and then try again. I bet this is the problem since I don't know if i have it

**Comment by asimenio-asteri** [comment]
ok so i just tried the document pdf again, this time it says
VIEW command invoked.
[16:37:24] Found root file from active editor: c:\Users\daver\Downloads\Cryptos\model\testttt.tex
[16:37:24] Root file remains unchanged from: c:\Users\daver\Downloads\Cryptos\model\testttt.tex.
[16:37:24] Cannot find PDF file c:\Users\daver\Downloads\Cryptos\model\testttt.pdf

**Comment by jlelong** [comment]
You need to build (compile) the `.tex` file before viewing it.
https://github.com/James-Yu/LaTeX-Workshop/wiki/Compile
